<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="195608" started-at="2021-06-03T07:10:29Z" finished-at="2021-06-03T07:13:45Z">
    <groups>
    </groups>
    <test name="Test Chilis WebApp" duration-ms="195608" started-at="2021-06-03T07:10:29Z" finished-at="2021-06-03T07:13:45Z">
      <class name="MobileAutomation.MobileSeeTestWebAutomation.TestLoggedInOrder">
        <test-method status="PASS" signature="initialize()[pri:0, instance:MobileAutomation.MobileSeeTestWebAutomation.TestLoggedInOrder@2d9d4f9d]" name="initialize" is-config="true" duration-ms="37910" started-at="2021-06-03T12:40:29Z" finished-at="2021-06-03T12:41:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method status="FAIL" signature="testLoggedInOrderFeature()[pri:0, instance:MobileAutomation.MobileSeeTestWebAutomation.TestLoggedInOrder@2d9d4f9d]" name="testLoggedInOrderFeature" duration-ms="150450" started-at="2021-06-03T12:41:07Z" finished-at="2021-06-03T12:43:38Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at Resources.Base.tearDown(Base.java:68)
	at Resources.FunctionalComponents.continueToPayment(FunctionalComponents.java:1024)
	at MobileAutomation.MobileSeeTestWebAutomation.TestLoggedInOrder.testLoggedInOrderFeature(TestLoggedInOrder.java:38)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLoggedInOrderFeature -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:MobileAutomation.MobileSeeTestWebAutomation.TestLoggedInOrder@2d9d4f9d]" name="tearDown" is-config="true" duration-ms="7167" started-at="2021-06-03T12:43:38Z" finished-at="2021-06-03T12:43:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- MobileAutomation.MobileSeeTestWebAutomation.TestLoggedInOrder -->
    </test> <!-- Test Chilis WebApp -->
  </suite> <!-- Suite -->
</testng-results>
